index.js

```diff
@@ -81,6 +81,10 @@ is.promise = x => {
 		);
 };
 
+is.generator = x => x && typeof x.next === 'function' && typeof x.throw === 'function';
+
+is.generatorFunction = x => x.constructor.name === 'GeneratorFunction';
```

Comment:
This fail throw on anything other than functions. You need to do:

```js
typeof x === 'function' && x.constructor && x.constructor.name === 'GeneratorFunction';
```

Reply:
But would be simpler (and probably safer) to just do `getObjectType(x) === 'GeneratorFunction'`